test/parallel/test-child-process-fork-closed-channel-segfault.js

```diff
@@ -31,6 +31,16 @@ const server = net
   .listen(0, function() {
     const worker = cluster.fork();
 
+    worker.on('error', function(err) {
+      if (
+        err.code !== 'ECONNRESET' &&
+        err.code !== 'ECONNREFUSED' &&
+        err.code !== 'EMFILE'
+      ) {
+        throw err;
+      }
```

Comment:
If I am not mistaken the test actually does not work in case one of these errors pops up. So instead of ignoring the error, I would like to skip the test with a message. That way there is at least some kind of notification that the test actually did not work as expected.

Reply:
@BridgeAR : the sole purpose of the test is to make sure the `channel` field of a closed worker process is propery nullified (ref: https://github.com/nodejs/node/issues/2847).

This is asserted after the [first send](https://github.com/gireeshpunathil/node/blob/fb5ac326f7b6c740b2cc62ebc8180fe97f948e88/test/parallel/test-child-process-fork-closed-channel-segfault.js#L74) causes the worker to [exit](https://github.com/gireeshpunathil/node/blob/fb5ac326f7b6c740b2cc62ebc8180fe97f948e88/test/parallel/test-child-process-fork-closed-channel-segfault.js#L14) and subsequently called back in the worker's [closure callback](https://github.com/gireeshpunathil/node/blob/fb5ac326f7b6c740b2cc62ebc8180fe97f948e88/test/parallel/test-child-process-fork-closed-channel-segfault.js#L64)

The race comes after: the second request can fail through a variety of reasons based on the state of the worker and the server, and their failure or success is immaterial to the test.

Hope this clarifies.

Reply:
If the error does not have any influence on the actual thing we want to test: great.